python/fast_mlsirm/estimators/marginal.py

```
@@ -1720,9 +1720,10 @@ def estep(current_params):
     converged = False
     final_delta = np.inf
     stopping_tolerance = float(tol * (1.0 + abs(ll)))
+    y_mask = (y[:, :, None] == np.arange(k_cat)).astype(np.float64)
     for it in range(1, max_iter + 1):
```

Ruff (0.16.2)

[warning] Loop control variable `it` not used within loop body

(B007)